hmda-regulator/src/main/scala/hmda/regulator/query/RegulatorComponent.scala

```diff
@@ -152,6 +152,7 @@ trait RegulatorComponent {
     def agency = column[Int]("agency")
     def totalLines = column[Int]("total_lines")
     def taxId = column[String]("tax_id")
+    def submissionId = column[String]("tax_id")
```

Comment:
This doesn't seem right. If we're adding the submissionId, we should pass the submission id.

Reply:
Oops, copy paste error, on it.